src/main/java/seedu/address/storage/JsonAdaptedStep.java

```diff
@@ -0,0 +1,48 @@
+package seedu.address.storage;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.step.Step;
+
+/**
+ * Jackson-friendly version of {@link Step}.
+ */
+class JsonAdaptedStep {
```

Comment:
If the JsonAdaptedStep is only storing one piece of information (step description), should we just do without the property name "step description"? It seems more similar to JsonAdaptedTag than JsonAdaptedPerson.

Reply:
That makes sense. It is weird though, since we don't have a StepDescription class

Reply:
Perhaps we should abstract the String out to a StepDescription? Would that be redundant

Reply:
yeah, tags also don't have a TagValue class